tokens/pickle.svg

```diff
@@ -0,0 +1,7 @@
.4.492 0 .583-.243.384-1.033-.204-.815-.093-1.077.526-1.239.645-.169.745-.446.563-1.566-.18-1.114-.093-1.362.478-1.362.543 0 .653-.24.494-1.075-.266-1.389.031-2.131.743-1.858.457.176.517-.134.313-1.625-.218-1.589-.156-1.842.453-1.842.601 0 .667-.241.444-1.635-.231-1.442-.162-1.661.586-1.856.836-.219.879-2.324.089-4.384-.132-.344.08-.651.532-.769.733-.192.75-1.857.659-64.276-.094-65.471-.065-64.005-1.471-74.28-.332-2.426-1.56-7.576-2.241-9.4l-2.009-5.2c-.175-.44-.644-1.79-1.042-3-2.815-8.549-10.462-18.519-21.92-28.578-2.402-2.109-7.997-6.222-8.464-6.222-.097 0-.847-.409-1.668-.91s-2.212-1.151-3.092-1.446-2.738-1.056-4.128-1.691-2.749-1.153-3.02-1.153-.663-.166-.872-.368c-.441-.429-4.849-1.973-7.18-2.516-7.406-1.726-10.364-2.084-22.348-2.706-4.592-.238-8.675-.608-9.074-.822s-1.178-.388-1.732-.388c-.997 0-4.715-.868-6.646-1.552-.55-.194-2.08-.682-3.4-1.083s-2.571-.872-2.78-1.047-.623-.318-.921-.318c-1.191 0-12.413-5.804-13.873-7.176-.483-.453-1.001-.824-1.153-.824-.937 0-5.537-3.534-10.414-8-3.687-3.376-9.466-10.622-11.644-14.6-.602-1.1-1.315-2.27-1.585-2.6s-1.288-2.263-2.261-4.295c-1.935-4.042-3.914-5.437-3.476-2.451m22.7 58.342c4.629 2.362 5.073 6.755 1.174 11.617-4.843 6.038-4.464 12.827 1.078 19.34 3.625 4.258.839 10.018-4.845 10.018-2.173 0-4.929-1.361-6.109-3.017-.28-.393-.74-.949-1.022-1.234s-.864-1.15-1.291-1.92-.872-1.49-.986-1.6c-.943-.905-2.266-5.365-2.782-9.375-.469-3.644.033-10.317.843-11.205.191-.209.348-.659.349-1 .003-1.632 3.713-7.675 6.06-9.873 2.794-2.618 4.881-3.103 7.531-1.751m32.046 34.748c2.43 2.738 2.442 4.858.048 8.238-4.342 6.13-4.797 16.134-.883 19.418 5.677 4.764.617 14.643-5.848 11.418-1.687-.842-4.956-4.149-4.956-5.015 0-.221-.149-.403-.332-.403-.499 0-3.001-5.256-4.013-8.428-.589-1.85-.59-11.291 0-13.144 1.726-5.423 3.569-8.462 7.452-12.287 2.442-2.406 6.298-2.314 8.532.203m-32.046 31.252c4.622 2.359 5.07 6.725 1.19 11.617-3.156 3.981-4.144 8.581-2.767 12.889.648 2.028 1.209 2.991 3.437 5.898 3.539 4.617 2.418 9.034-2.637 10.396-3.805 1.024-7.546-1.174-10.251-6.024-.421-.755-.859-1.462-.973-1.572-.933-.897-2.265-5.359-2.769-9.279-.459-3.568.045-10.452.827-11.301.192-.209.351-.659.352-1 .003-1.632 3.713-7.675 6.06-9.873 2.794-2.618 4.881-3.103 7.531-1.751m30.371 33.347c3.589 1.701 4.424 5.906 1.878 9.451-3.582 4.988-4.941 12.549-2.913 16.206.367.66.667 1.391.669 1.625s.322.773.714 1.2 1.234 1.464 1.872 2.306c3.734 4.934-1.823 12.383-7.228 9.687-1.687-.842-4.956-4.149-4.956-5.015 0-.221-.149-.403-.332-.403-.499 0-3.001-5.256-4.013-8.428-.59-1.851-.59-11.292 0-13.144 3.243-10.17 9.285-15.865 14.309-13.485m-30.371 32.653c4.622 2.359 5.07 6.725 1.19 11.617-3.156 3.981-4.144 8.581-2.767 12.889.65 2.036 1.217 3.005 3.447 5.898 3.393 4.401 2.399 9.051-2.265 10.601-3.843 1.276-7.711-.99-10.633-6.229-.421-.755-.855-1.462-.965-1.572-.45-.45-1.6-3.277-1.6-3.934 0-.388-.158-.877-.35-1.086-1.252-1.359-1.252-14.201 0-15.56.192-.209.351-.659.352-1 .003-1.632 3.713-7.675 6.06-9.873 2.794-2.618 4.881-3.103 7.531-1.751" fill="#0c540c" />
```

Comment:
The icon is super-big, this won't work, unfortunately. Please try reducing it to 5-7 KB.